ui/components/ui/text-field/text-field.component.js

```diff
@@ -241,6 +241,8 @@ const TextField = ({
     inputProps.InputProps.inputProps['data-testid'] = dataTestId;
   }
 
+  // inputProps.InputProps.inputProps.step = '0.1';
+
```

Comment:
Nit: leftover comment

```suggestion
```

Reply:
Good catch — removed in [c457767](https://github.com/MetaMask/metamask-extension/pull/19653/commits/c457767c0c8c2a00dfb1b099d2d42813b3d440e3).